src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -1070,37 +1020,48 @@ public static function update(array $item): int
         return $item['revision_id'];
     }
 
-    /**
-     * Update an existing category
-     *
-     * @param array       $item The new data.
-     * @param array $meta The new meta-data.
-     *
-     * @return int
-     */
-    public static function updateCategory(array $item, array $meta = null): int
-    {
-        // get database
-        $database = BackendModel::getContainer()->get('database');
+    public static function updateCategory(
+        array $item,
+        ?array $meta = null
```

Comment:
why the additional useless `?` again?

Reply:
Fixed in e1a30444f2858439c12fb88659292f551b1e924a